### PR TITLE
Fix/ADF-577/Disable unavailable tabs in the 'Get started with TAO' popup

### PR DESCRIPTION
**Pull request by shpran**
**Relates to:** _[ADF-577](https://oat-sa.atlassian.net/browse/ADF-577)_

---

**Changes:** Additional extensions tabs will be disabled if user does not have access.

---

**How to test:**
1. Create user with `Item Content Creator` role;
2. Login as this user;
3. Take a look at the "Get started with TAO" popup.

**Previous result:** _`Assets` tab is active and can be clicked._
**Current result:** _`Assets` tab is disabled and cannot be clicked._

### CODE REVIEW

File: actions/Home.php [hunk 1]
```diff
@@ -15,72 +15,65 @@
...
+    /** This action renders the template used by the splash screen popup */
+    public function splash(): void
     {
-
-        //the list of extensions the splash provides an explanation for.
+        // The list of extensions the splash provides an explanation for.
         $defaultExtIds = ['items', 'tests', 'TestTaker', 'groups', 'delivery', 'results'];
 
-        //check if the user is a noob
+        // Check if the user is a noob
```
Comment (gabrielfs7) [inline actions/Home.php:40]: Thanks we have this comment to explain :)

File: views/templates/splash.tpl [hunk 2]
```diff
@@ -92,7 +92,12 @@
                     foreach(get_data('additionalExtensions') as $extension):
                         ?>
                         <?php if(!$moreShowed) echo '<span class="more">More:</span>';?>
-                        <a href="#" class="module new-module" data-module-name="<?=$extension['id']?>" data-url="<?=_url('index', 'Main', 'taoCe', array('structure' => $extension['id'], 'ext' => $extension['extension']))?>">
+                        <a
+                            href="#"
+                            class="module new-module<?php if(!$extension['enabled']): ?> disabled<?php endif ?>"
```
Comment (gabrielfs7) [inline views/templates/splash.tpl:97]: Do you think it is cleaner?

```suggestion
                            class="module new-module<?= !$extension['enabled'] ? 'disabled' : ''; ?>"
```

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: views/templates/splash.tpl (hunk 2 above)
Comment (habemuscode) [reply views/templates/splash.tpl:97]: Agree its cleaner :)

**Comment by habemuscode** [comment]
Added my changes

imagen

File: views/templates/splash.tpl (hunk 2 above)
Comment (shpran) [reply views/templates/splash.tpl:97]: Not agree with short tags 🙂
https://wiki.php.net/rfc/deprecate_php_short_tags

@gabrielfs7

Comment (gabrielfs7) [reply views/templates/splash.tpl:97]: Check you link for this `The <?= short tag is unaffected because as of PHP 5.4 it is always available.`

**Review by habemuscode — APPROVED** [review]
LGTM

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Pull request merged**